design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,147 @@
+type Mutation {
+    createNegotiableQuote(
+        cart_id: ID! @doc(description: "Cart ID")
+        quote_name: String @doc(description: "Quote name")
+        comment: String @doc(description: "Comment")
+        files: [FileInput!] @doc(description: "Attached files")
+    ): RequestNegotiableQuoteOutput! @doc(description:"Generates negotiable quote request")
+
+    setShippingMethodsOnNegotiableQuote(
+        cart_id: ID! @doc(description: "Cart ID")
+        shipping_methods: [String!]! @doc(description: "Input for shipping method")
+    ): SetShippingMethodsOnNegotiableQuoteOutput @doc(description:"Set negotiable quote shipping method")
+}
+
+type Customer {
+    negotiable_quote (
+        id: ID!): NegotiableQuote @doc(description: "Get negotiable quote of a customer")
+
+    negotiable_quotes(
+        page_size: Int = 20,
+        current_page: Int = 1,
+        filter: NegotiableQuoteFilterInput
```

Comment:
It does not seem to be possible to filter in the current Luma implementation. Why the filtration capability was added? 
@mbrinton01 do we need it?

Reply:
We currently have advanced search for Orders in the storefront (in 2.4) with more basic implementation for Quotes. It is not currently supported to filter the Quotes grid, although we intend to add this capability in a similar way to Order search.